### PR TITLE
Add support for Emoji 13

### PR DESCRIPTION
**Pull request by mauscoelho**

### CODE REVIEW

**Review by RalfNieuwenhuizen — COMMENTED** [review]
Love those keywords

**Comment by aminmahrami** [comment]
Did you update the `yarn.lock` too? Maybe that's why emojis are not shown.

File: src/Emoji.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 import { View, Text, TouchableOpacity, StyleSheet, Image, Platform } from 'react-native';
 import _ from 'lodash';
 
-const EMOJI_DATASOURCE_VERSION = '4.0.4';
+const EMOJI_DATASOURCE_VERSION = '6.0.0';
```
Comment (mauscoelho) [inline src/Emoji.js:6]: 👍

**Comment by aminmahrami** [comment]
😢

**Comment by mauscoelho** [comment]
@aminmahrami  looks like nobody is going to merge it so I closed 😂

**Comment by AlessandroAries** [comment]
@mauscoelho are you going working at Relive again 😜

**Pull request closed without merging**